### ISSUE TITLE
[BUG] Increase resource for POD grafana

### ISSUE THREAD

**Issue by w13915984028** [harvester/harvester#2361]
**Describe the bug**

In a single-node cluster, with one guest VM, the Grafana POD's memory usage is close to limit. It is necessary to increase the limits value.
![image](https://user-images.githubusercontent.com/31133476/172632067-99001bc0-ef53-43ca-a8fe-ad8375d94320.png)

```
                "resources": {
                    "limits": {
                        "cpu": "200m",
                        "memory": "200Mi"
                    },
                    "requests": {
                        "cpu": "100m",
                        "memory": "100Mi"
                    }
                },
```

**To Reproduce**
1. Install Harvester with master-head
2. Create a few VMs
3. Check the memory usage of POD grafana from Harvester metrics.

**Expected behavior**

Has a proper memory limit for POD grafana.

**Support bundle**

**Environment**
 - Harvester ISO version:  V1.0.1
 - Underlying Infrastructure (e.g. Baremetal with Dell PowerEdge R630): KVM based Cluster

**Additional context**

**Comment by DaiYuzeng** [comment]
@w13915984028 Hi! I realized there is the `area/ui` label. 

Users cannot modify the configuration of Grafana on GUI now, they can just modify configurations below:
![image](https://user-images.githubusercontent.com/15041915/183551382-58899795-ac48-47f2-9d71-e3e195ffbe6a.png)

Should we display the resources configurations of Grafana on GUI?

**Comment by w13915984028** [comment]
@DaiYuzeng  I would like to suggest that we add similar resource configuration for Grafana POD in Harvester GUI.
This issue is in lower priority than others.

**Comment by noahgildersleeve** [comment]
Tested in master-4ea12f4f-head. It looks like the limit was increased to the following
![Screenshot_20220831_130009](https://user-images.githubusercontent.com/83787952/187771896-255f42a5-867e-4aa5-9460-038816515c27.png)
This was what I saw with `describe pod rancher-monitoring-grafana-647fd577f4-qfks2`. The memory usage seemed better after creating a few VMs.
![Screenshot_20220831_125525](https://user-images.githubusercontent.com/83787952/187772092-b2459361-44c1-466a-b2e4-fb9dd7110faa.png)
Closing as fixed.
